airflow/cli/cli_parser.py

```diff
@@ -263,6 +263,9 @@ def string_lower_type(val):
 )
 
 # list_dag_runs
+ARG_DAG_ID_REQ_FLAG = Arg(
+    ("-d", "--dag-id"), required=True, help="The id of the dag"
+)  # convert this to a positional arg in Airflow 3
```

Comment:
```suggestion
)  # TODO: convert this to a positional arg in Airflow 3
```

so it shows up in the TODO list in editors or via regex